airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -465,6 +467,7 @@ def _stream_properties(self, fp, empty_schema: bool = False, read_sample_chunk:
             for field in fields
         }
 
+    @backoff.on_exception(backoff.expo, ConnectionResetError, on_backoff=backoff_handler, max_tries=MAX_TRIES, max_time=MAX_TIME_BEFORE_GIVE_UP)
```

Comment:
Aren't we already retrying on error [here](https://github.com/airbytehq/airbyte/blob/c95eb459cf674a8b3a199c2e405e30c7b68e9a7f/airbyte-integrations/connectors/source-file/source_file/client.py#L112)? Are we saying the we don't mind retrying `MAX_TRIES * MAX_TRIES` (100 times in for this specific example) in the worst case. It seems aggressive to me but I don't fully understand the issue